cogs/spoilers.py

```diff
@@ -0,0 +1,117 @@
+"""Spoilers cog
+Filters out messages that start with a certain prefix, and store them for
+later retrieval.
+"""
+
+import os
+import logging
+import discord
+from discord.ext import commands
+from cogs.utils.dataIO import dataIO
+
+from cogs.utils import config
+
+#Global variables
+KEY_MESSAGE = "message"
+KEY_AUTHOR_ID = "authorid"
+KEY_AUTHOR_NAME = "author"
+LOGGER = None
+PREFIX = "spoiler"
+SAVE_FOLDER = "data/lui-cogs/spoilers/" #Path to save folder.
+SAVE_FILE = "settings.json"
+
+def checkFolder():
+    """Used to create the data folder at first startup"""
+    if not os.path.exists(SAVE_FOLDER):
+        print("Creating " + SAVE_FOLDER + " folder...")
+        os.makedirs(SAVE_FOLDER)
+
+def checkFiles():
+    """Used to initialize an empty database at first startup"""
+    theFile = SAVE_FOLDER + SAVE_FILE
+    if not dataIO.is_valid_json(theFile):
+        print("Creating default spoilers settings.json")
+        dataIO.save_json(theFile, {})
+
+class Spoilers: # pylint: disable=too-many-instance-attributes
+    """Store messages for later retrieval."""
+
+    #Class constructor
+    def __init__(self, bot):
+        self.bot = bot
+
+        #The JSON keys for the settings:
+        checkFolder()
+        checkFiles()
+        self.settings = config.Config("settings.json",
+                                      cogname="lui-cogs/spoilers")
+        self.messages = self.settings.get("messages") if not None else {}
+
+    @commands.command(name="spoiler", pass_context=True)
+    async def spoiler(self, ctx, *, msg):
+        """Create a message spoiler.
+        Checks to see if the message contains the prefix, and if it does, it saves it
+        for later retrieval.

```

Comment:
I guess this comment is out of date, since you made this into a command I think.